examples/calling-apis/spa-with-backend-api/react-hono-ai-sdk/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "chatbot",
+  "version": "0.4.0",
+  "description": "A monorepo template built with npm, Hono, Vite, and React",
+  "author": {
+    "name": "Auth0 Inc.",
+    "url": "https://auth0.com"
+  },
+  "license": "Apache-2.0",
+  "packageManager": "npm@10.9.2",
+  "workspaces": [
+    "./server",
+    "./client",
+    "./shared",
+    "../../../../packages/ai",
+    "../../../../packages/ai-vercel"
```

Comment:
minor mention about this: preference here would be to keep these dependent packages in the build of this spa-with-backend-api app. This ensures automatic discovery and re-building of local changes. As a side note, i'm noting that b/c the other examples are not included in the root [package.json](https://github.com/auth0-lab/auth0-ai-js/blob/main/package.json#L26-L28) workspace definition, the `*` references in the other examples are not truly symlinked and are just local copies. 
```
  "workspaces": [
    "packages/*"
  ],
```

That said, we could always just reference a deployed version, but preference here is to keep the local workspace version as the others. This won't be necessary once symlinks are incorporated in the root.